compute/admin_guide/configure/agentless-permission-templates.adoc

```diff
@@ -0,0 +1,68 @@
+== Agentless Permission Templates
+
+Agentless scanning requires specific permissions to work on your self-hosted deployments.
+Different platforms have different permission requirements. 
+To make applying the right permissions easier, you can download permission templates for your specific platforms.
+The following sections explain the different permission templates and what they entail.
+
+=== AWS
+
+You need to download three permission templates as JSON files for AWS cloud account scanning. The templates are needed to enable the different ways accounts are scanned: independent scanning, scan hub account, and target account.
+
+==== Independent Scanning
+
+All scanning happens within the same account.
+This account is independent of other accounts.
+The account requires all  needed permissions to create snapshots, create scanners, etc. 
+The permission template you need to download ends in `_target_user_permissions.json`.
+
+==== Scan Hub Account
+
+This account is the hub account where you want all account scanning to happen.
+It scans not just the hosts in this account, but it also scans the hosts in other cloud accounts.
+This account requires the permissions needed to run Prisma Cloud Scanners and to access to all other cloud accounts.
+These permissions allow the hub account to see and transfer host snapshots from other accounts to this account. 
+The permission template you need to download ends in `_hub_user_permissions.json`.
+
+==== Target Account
+
+This is the account that a hub account scans.
+The permissions needed in target accounts are less than those required for independent scanning, but agentless still needs specific permissions to enable the hub account to scan a target account.
+The permission template you need to download ends in `_hub_target_user_permissions.json`.
```

Comment:
"Apply the permission template that ends with:  `_hub_target_user_permissions.json` to the accounts that are being scanned bu Hub account.